x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -52,6 +52,17 @@ export const TRACK_CURRENT_LAYER_STATE = 'TRACK_CURRENT_LAYER_STATE';
 export const ROLLBACK_TO_TRACKED_LAYER_STATE = 'ROLLBACK_TO_TRACKED_LAYER_STATE';
 export const REMOVE_TRACKED_LAYER_STATE = 'REMOVE_TRACKED_LAYER_STATE';
 export const SET_TOOLTIP_STATE = 'SET_TOOLTIP_STATE';
+export const UPDATE_DRAW_STATE = 'UPDATE_DRAWMODE';
+
+export const DRAW_STATE_TYPE = {
+  NONE: 'NONE',
+  ACTIVE: 'ACTIVE'
+};
+
+export const DRAW_STATE_DRAW_TYPE = {
```

Comment:
`DRAW_STATE_DRAW_TYPE` is a weird name having draw in there twice. How about just renaming to `DRAW_TYPE`